wrappers/python/indy/anoncreds.py

```diff
@@ -1007,7 +1009,7 @@
         {
             "name": string,
             "version": string,
```

Comment:
Is there an example of version somewhere

Reply:
This `version` field is for apps. It can be any for now. `Proof Request` (structure) versioning isn't implemented yet.